### ISSUE TITLE
Towards V0.0.1

### ISSUE THREAD

**Issue by leios** [leios/Fable.jl#20]
Very far from any official release, but here is just a grab bag of things we need to complete before the first tag:

* [x] `fractal_flame!(...)` -> `run(...)`, which will run an Fractal Executable with kwarg for fractal flame. This allows for additional solvers and settles the API a bit.
* [x] Objects should primarily use kwargs, so `create_rectangle(; ...)`, this makes it easier to create basic shapes without fussing about what the arg order is. This should also be true for `FractalExecutable`s
* [x] Layers should be writable in any order. The bottom layer could, for example, be a `FractalLayer`
* [x] Adding examples for:
    * [x] Layering and each layer type
    * [x] Shapes and each shape type
* [x] Feature presentation branch
* [x] Layers should be addable at specific coordinates. ie, add a color frame at position (10, 10) or something.i
* [x] Add CPU tests to CI. GPU tests are a pain, so maybe they are skipped for this release
* [x] Add boolean `solve` to each layer, so we do not need to regenerate layers if they are already done
* [x] Documentation (#40)
* [x] Lollipeople (#16)
* [x] Fae in Registry.jl (register in comments when everything else is done)

**Comment by leios** [comment]
@JuliaRegistrator register

**Comment by leios** [comment]
@JuliaRegistrator register

**Comment by leios** [comment]
I cannot put Fae / Fable into the Registries until #68 is fixed.

**Comment by leios** [comment]
Ok, it looks like 0.0.1 is done. People can use it and I'll be making the feature presentation demo later this week.

**Comment by leios** [comment]
@JuliaRegistrator register
